ui/src/views/Weights.vue

```diff
@@ -2,55 +2,53 @@
 import axios from "axios";
 
 export default {
-  data() {},
+  data() {
+    return {
+      clientId: [],
+      clientWeights: [],
+      client_name: " ",
+    };
+  },
+
   mounted() {
     this.getWeights();
   },
 
   methods: {
-    getWeights() {
+    getWeights(clientId) {
       axios
-        .get(`${import.meta.env.VITE_API_URL}clients_weights/${clientId}`)
-        .then((response) => {
-          this.clients = response.data;
-        });
-    },
-
-    addWeight() {
-      const requestBody = {
-        client_name: this.newClientName,
-        birth_day: this.newClientBirthDate,
-      };
-      axios
-        .post(
-          `${import.meta.env.VITE_API_URL}clients_weights/${clientId}`,
-          requestBody
+        .get(
+          `${import.meta.env.VITE_API_URL}clients_weights/${
+            this.$route.params.clientId
+          }`
         )
         .then((response) => {
-          console.log(response);
-          this.getWeights();
-          this.newClientName = "";
-          this.newClientBirthDate = "";
+          this.clientWeights = response.data;
         });
     },
   },
 };
-
-// deleteWeight(clientId) {
-//   console.log("delete");
-//   console.log(clientId);
-//   axios
-//     .delete(`${import.meta.env.VITE_API_URL}clients/${clientId}`)
-//     .then((response) => {
-//       console.log(response);
-//       this.getClients();
-//     });
-// },
 </script>
 <template>
   <div id="app">
-    <h1>Go to the gym!</h1>
-    <table></table>
+    <h1>Weights</h1>
+    <table>
+      <tr>
+        <th>Weight</th>
+        <th>Date</th>
+      </tr>
+      <tr v-for="weight in clientWeights">
+        <td>
+          {{ weight.weight }}
+        </td>
+        <td>{{ weight.date }}</td>
+
+        <td><button @click="deleteWeight(client.id)">🗑</button></td>
```

Comment:
I think clicking this would cause an error since you don't have a `deleteWeight` function